docs/authoring.md

```diff
@@ -86,78 +86,140 @@ To make your component available as a NuGet package, it is important to include
 
-Follow these steps to add an application manifest file to your project:
+  - For **packaged** C++ apps: The app consumer needs to add `ActivatableClass` registration entries to the `Package.appxmanifest` package manifest file like below:
 
-1. In Visual Studio, right click on the project node under **Solution Explorer** and click **Add -> New Item**. Search for the **Text File** template and name your file `YourNativeApp.exe.manifest`.
+      ``` xml
+      <Extensions>
+        <Extension Category="windows.activatableClass.inProcessServer">
+          <InProcessServer>
+            <Path>WinRT.Host.dll</Path>
+            <ActivatableClass ActivatableClassId="MyAuthoredComponent.Class1" ThreadingModel="both" />
+            <ActivatableClass ActivatableClassId="MyAuthoredComponent.Class2" ThreadingModel="both" />
+          </InProcessServer>
+         </Extension>
+      </Extensions>
+      ```
 
-2. Add your activatable class registrations to the manifest file. Below is an example with a C#/WinRT component `MyAuthoredComponent` and activatable classes `WinRTComponent.Class*`. Note that `WinRT.Host.dll` must be specified as the file name.
+      See the [Simple C#/WinRT component sample](../src/Samples/AuthoringDemo) (specifically the WinUI3CppApp) for an example.
 
-      ```xml
-      <?xml version="1.0" encoding="utf-8"?>
-      <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
-      <assemblyIdentity version="1.0.0.0" name="CppConsoleApp"/>
-      <file name="WinRT.Host.dll">
-      <activatableClass
-            name="MyAuthoredComponent.Class1"
-            threadingModel="both"
-            xmlns="urn:schemas-microsoft-com:winrt.v1" />
-      <activatableClass
-            name="MyAuthoredComponent.Class2"
-            threadingModel="both"
-            xmlns="urn:schemas-microsoft-com:winrt.v1" />
-      </file>
-      </assembly>
-      ```
+  - For **unpackaged C++ apps**: The app consumer needs to create an [application manifest file](https://docs.microsoft.com/windows/win32/sbscs/application-manifests) (also known as a "fusion manifest") that follows the naming: `<MyNativeAppName>.exe.manifest`. See this [sample manifest file](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/AuthoringDemo/CppConsoleApp/CppConsoleApp.exe.manifest) for an example.
 
-3. Modify the project to include the manifest file in the output when deploying the project. Right-click on the file in **Solution Explorer**, select **Properties**, and set the **Content** property to **True** using the drop-down arrow on the right.
+      Follow these steps to add an application manifest file to your project:
 
-### Consuming from C# applications
+      1. In Visual Studio, right click on the project node under **Solution Explorer** and click **Add -> New Item**. Search for the **Text File** template and name your file `<MyNativeAppName>.exe.manifest`.

```

Comment:
Wasn't there also a manifest file template?  Is that not better?

Reply:
Don't think so, this is how we've always done it

Reply:
there is a manifest item template, but only for managed projects, for some reason....
![image](https://user-images.githubusercontent.com/22989529/157344614-8ddf074d-b44c-47cf-bdd2-0bd8d32d7807.png)